tests/unit/variant_browser.js

```diff
@@ -209,6 +209,22 @@ describe('Consequence Type Tree', function(){
 
         expect(conseqTypeFilter._getConsequenceTypeTreeFormat(testObjectBefore)).to.deep.equal(expectedObjectAfter);
     });
+});
 
+describe('VEP version Notification Text', function(){
+    var annotationPanel = new ClinvarAnnotationPanel();
+    var testObjectBefore =  {
+        "vepVersion": "89",
+        "cacheVersion": "35",
+        "defaultVersion": true
+    };
 
+    it('should return empty for chircus_10', function(){
+        expect(annotationPanel.getVepNotificationText('chircus_10',testObjectBefore)).to.deep.equal('');
+    });
+    
+    it('should return  "Variant Effect Predictor (VEP) v89 annotation against the Ensembl Genomes v35 geneset" for aaegypti_aaegl3', function(){
+        var expectedObjectAfter = "Variant Effect Predictor (VEP) v89 annotation against the Ensembl Genomes v35 geneset.";
+        expect(annotationPanel.getVepNotificationText('aaegypti_aaegl3',testObjectBefore)).to.deep.equal(expectedObjectAfter);
```

Comment:
This test will start failing as soon as the web service returns a newer version of the annotation. The web service response should be mocked if possible.